src/App.tsx

```diff
@@ -20,12 +26,14 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => setQuery(event.target.value)}
```

Comment:
better make function outside render, not to create it every time we changing state